<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>391</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>qtcsdr</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QMySpectrumWidget" name="widgetFFT" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="widgetControls" native="true">
      <property name="minimumSize">
       <size>
        <width>603</width>
        <height>159</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>159</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-4</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Frequency:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="toggleAM">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>0</y>
         <width>30</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>AM</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>25</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Offset:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinFreq">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>0</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>1999999999</number>
       </property>
       <property name="singleStep">
        <number>1000</number>
       </property>
       <property name="value">
        <number>89500000</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinOffset">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>-1200000</number>
       </property>
       <property name="maximum">
        <number>1200000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>200000</number>
       </property>
      </widget>
      <widget class="QPushButton" name="toggleWFM">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>0</y>
         <width>30</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>WFM</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="toggleUSB">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>0</y>
         <width>30</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>USB</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="toggleLSB">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>0</y>
         <width>30</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>LSB</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="toggleRun">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Power</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="toggleNFM">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>0</y>
         <width>30</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NFM</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinCenter">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>1999999999</number>
       </property>
       <property name="singleStep">
        <number>1000</number>
       </property>
       <property name="value">
        <number>89300000</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Center:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboDirectSamp">
       <property name="geometry">
        <rect>
         <x>513</x>
         <y>0</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Quadrature sampling</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Direct sampling (I branch)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Direct sampling (Q branch)</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="toggleTransmit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>50</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PTT</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>39</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Samp. rate:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboSampRate">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>2400000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2048000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1200000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1024000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>960000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>480000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>240000</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="toggleCW">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>0</y>
         <width>30</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CW</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="toggleCWR">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>0</y>
         <width>30</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CWR</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumberPanel">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>30</y>
         <width>231</width>
         <height>71</height>
        </rect>
       </property>
       <property name="smallDecimalPoint">
        <bool>true</bool>
       </property>
       <property name="digitCount">
        <number>7</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBarMeter">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>110</y>
         <width>231</width>
         <height>15</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="format">
        <string>%p</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAB">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>130</y>
         <width>40</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>A/B</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonA2B">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>130</y>
         <width>40</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>A&gt;B</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonABS">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>130</y>
         <width>40</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>A&lt;&gt;B</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonSplit">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>130</y>
         <width>40</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Split</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonRIT">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>130</y>
         <width>40</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RIT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonLock">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>130</y>
         <width>40</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Lock</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>110</y>
         <width>48</width>
         <height>15</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Meter</string>
       </property>
      </widget>
      <widget class="QSlider" name="verticalSlider">
       <property name="geometry">
        <rect>
         <x>165</x>
         <y>10</y>
         <width>20</width>
         <height>100</height>
        </rect>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="verticalSlider_2">
       <property name="geometry">
        <rect>
         <x>187</x>
         <y>9</y>
         <width>20</width>
         <height>100</height>
        </rect>
       </property>
       <property name="value">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="verticalSlider_3">
       <property name="geometry">
        <rect>
         <x>208</x>
         <y>9</y>
         <width>20</width>
         <height>100</height>
        </rect>
       </property>
       <property name="value">
        <number>75</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="verticalSlider_4">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>10</y>
         <width>20</width>
         <height>100</height>
        </rect>
       </property>
       <property name="value">
        <number>25</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="verticalSlider_5">
       <property name="geometry">
        <rect>
         <x>252</x>
         <y>10</y>
         <width>20</width>
         <height>100</height>
        </rect>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QMySpectrumWidget</class>
   <extends>QWidget</extends>
   <header>qmyspectrumwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
